presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/GeometryStateSerializerTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial.aggregation;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import com.facebook.presto.operator.aggregation.state.StateCompiler;
+import com.facebook.presto.plugin.geospatial.GeometryType;
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.block.BlockBuilder;
+import com.facebook.presto.spi.function.AccumulatorStateFactory;
+import com.facebook.presto.spi.function.AccumulatorStateSerializer;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class GeometryStateSerializerTest
+{
+    @Test
+    public void testSerializeDeserialize()
+    {
+        AccumulatorStateFactory<GeometryState> factory = StateCompiler.generateStateFactory(GeometryState.class);
+        AccumulatorStateSerializer<GeometryState> serializer = StateCompiler.generateStateSerializer(GeometryState.class);
+        GeometryState state = factory.createSingleState();
```

Comment:
I'm not sure we need these fine-grained unit tests, but if we keep them they need to test grouped state as well.